engine/classes/ElggCache.php

```diff
@@ -46,12 +41,8 @@ public function setVariable($variable, $value) {
 	 *
 	 * @return mixed The variable or null;
 	 */
-	public function getVariable($variable) {
-		if (isset($this->variables[$variable])) {
-			return $this->variables[$variable];
-		}
-
-		return null;
+	public function getVariable(string $variable): ?string {
```

Comment:
return type doesn't match the allowed variable type in the setter and also in the function docs